packages/account/src/Components/sample-credit-card-modal/sample-credit-card-modal.tsx

```diff
@@ -1,9 +1,21 @@
 import React from 'react';
 import { Modal, Text } from '@deriv/components';
-import { localize, Localize } from '@deriv/translations';
+import { Localize } from '@deriv/translations';
 import { getUrlBase } from '@deriv/shared';
 
-export const SampleCreditCardModal = ({ is_open, onClose }) => {
+type TSampleCreditCardModalProps = {
+    is_open: boolean;
+    onClose: () => void;
+};
+
+/**
+ * Display a modal with a sample credit card image and instructions on how to mask the card.
+ * @name SampleCreditCardModal
+ * @param is_open - boolean to determine if the modal should be open or not
+ * @param onClose - function to close the modal
+ * @returns React component
+ */
+export const SampleCreditCardModal = ({ is_open, onClose }: TSampleCreditCardModalProps) => {
```

Comment:
... => (<Modal... ?

you may also ignore these.

Reply:
Will remove the return statement